src/components/Order/Order.tsx

```diff
@@ -4,12 +4,15 @@ import styled from 'styled-components';
 import orderAction from '../../constants/orderAction';
 import orderOptions from '../../constants/orderOptions';
 import OrderType from '../../models/orderType';
+import {StorageUtils} from '../../utils/index';
 import {OrderProps, OrderState} from './index';
 import OrderAction from './OrderAction';
 import OrderButton from './OrderButton';
 import OrderChoiceButton from './OrderChoiceButton';
 import OrderOption from './OrderOption';
 
+const baseAssetStorage = StorageUtils('baseAsset');
```

Comment:
it's better to prefix all keys with smth mnemonic like `lwt` = Lykke web terminal. guess it can be done on storage utils